recipe/meta.yaml

```diff
@@ -26,10 +26,13 @@ requirements:
     - {{ posix }}patch  # [win]
   host:
     - ffmpeg
-    - gstreamer
-    - gst-plugins-base
+    - gstreamer {{ gstreamer }}
+    - gst-plugins-base {{ gstreamer }}
     - glib
     - zlib  # [win] # required for it to find z.lib for some reason
+  run:
```

Comment:
I think you can remove the "run" lines since there is a `run_exports` in the gstreamer packages.

You can if you want, remove the addition. When you brought up the issue I "fixed" the migrator, so it should be more consistent without the `{{ gstreamer }}` addition.

Reply:
Okay, let's see if it builds properly now without pinning the gstreamer version.
If it builds, I'll merge.

Before the migrator update, it complained about gst-plugins-base v1.18 being the wrong version.
There seems to be a hard dependency between those packages.